certified-connectors/Industrial App Store/apiDefinition.swagger.json

```diff
@@ -330,63 +330,57 @@
             "schema": {
               "type": "object",
               "properties": {
-                "datasource": {
+                "tagname": {
                   "type": "object",
                   "properties": {
-                    "tagname": {
+                    "TagName": {
+                      "type": "string",
+                      "description": "TagName"
+                    },
+                    "UtcSampleTime": {
+                      "type": "string",
+                      "description": "UtcSampleTime"
```

Comment:
@gsmcmurtrie Can you please provide valid description values.

Documentation - [Summary and Description values](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the description values in the swagger